r/tests/testthat/test-dplyr-collapse.R

```diff
@@ -142,21 +142,6 @@ test_that("Properties of collapsed query", {
     summarize(total = sum(int, na.rm = TRUE)) %>%
     mutate(extra = total * 5)
 
-  # print(tbl %>%
-  #   filter(dbl > 2) %>%
-  #   select(chr, int, lgl) %>%
-  #   mutate(twice = int * 2L) %>%
-  #   group_by(lgl) %>%
-  #   summarize(total = sum(int, na.rm = TRUE)) %>%
-  #   mutate(extra = total * 5))
-
-  #   # A tibble: 3 × 3
-  #   lgl   total extra
-  #   <lgl> <int> <dbl>
-  # 1 FALSE     8    40
-  # 2 TRUE      8    40
-  # 3 NA       25   125
```

Comment:
This is what `print(q)` would be, yeah? I wonder if we should keep the printed tibble output part here around as "this is what this actually looks like"?

Reply:
Added back